app/views/administrate/application/index.html.erb

```diff
@@ -21,7 +21,9 @@ It renders the `_table` partial to display details about the resources.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<% content_for(:title) { page.resource_name.pluralize.titleize } %>
+<% content_for(:title) do
+  page.resource_name.classify.constantize.model_name.human(default: page.resource_name.pluralize.titleize)
```

Comment:
So better not to do. For example, for the Russian language the forced pluralization it looks like this:

```
# config/locales/ru.yml
ru:
  activerecord:
    models:
      bet: Ставки
```

```
[1] pry(main)> Bet.model_name.human(default: 'Bets').pluralize
=> Ставкиs
```